ts/features/it-wallet/store/reducers/index.ts

```diff
@@ -0,0 +1,13 @@
+import { combineReducers } from "redux";
+import { Action } from "../../../../store/actions/types";
+import itwRequirements, { ItwRequirementsState } from "./itwRequirements";
+
+export type ItWalletState = {
+  itwRequirements: ItwRequirementsState;
```

Comment:
Suggest to remove itw from state variables so we have 
`{
features: {
  //...
    itWallet: {
      requirements: //...
    }
}`

```suggestion
  requirements: ItwRequirementsState;
```